proto/cosmos/params/query.proto

```diff
@@ -6,20 +6,23 @@ import "cosmos/params/params.proto";
 
 option go_package = "github.com/cosmos/cosmos-sdk/x/params/types/proposal";
 
-// Query creates service with Parameters as rpc
+// Query defines the gRPC querier service.
 service Query{
-    // Params queries all parameters of the params module
+    // Params queries a specific parameter of a module, given its subspace and key.
     rpc Params (QueryParamsRequest) returns (QueryParamsResponse) {}
 }
 
-// QueryParamsRequest is request type for the Query/Params RPC method
+// QueryParamsRequest is request type for the Query/Params RPC method.
 message QueryParamsRequest{
+    // subspace defines the module to query the parameter for.
     string subspace = 1;
 
+    // key defines the key of the parameter in the subspace.
     string key = 2;
 }
 
-// QueryParamsResponse is response type for the Query/Params RPC method
+// QueryParamsResponse is response type for the Query/Params RPC method.
 message QueryParamsResponse{
-    cosmos.params.ParamChange params = 1 [(gogoproto.nullable) = false];
+    // param defines the queried parameter.
+    cosmos.params.ParamChange param = 1 [(gogoproto.nullable) = false];
```

Comment:
note that i changed this from params to param.

Reply:
Why is that?

Reply:
Because the response only returns one param:

```proto
message ParamChange {
  option (gogoproto.goproto_stringer) = false;

  string subspace = 1;
  string key      = 2;
  string value    = 3;
}
```

Reply:
Ahh right! Thanks :)